scaffold/review.js

```diff
@@ -0,0 +1,45 @@
+const inquirer = require('inquirer');
+const writeData = require('./writeData.js')
```

Comment:
We have to use commonjs module requirements?

Reply:
i _think_ so because idk if node automatically has this or not. we can try it with import and see if it works. might need to lock the node version somewhere